decidim-admin/spec/factories.rb

```diff
@@ -3,3 +3,5 @@
 require "decidim/core/test/factories"
 require "decidim/participatory_processes/test/factories"
 require "decidim/verifications/test/factories"
+require "decidim/conferences/test/factories"
+require "decidim/assemblies/test/factories"
```

Comment:
I think these 2 lines needs to be removed. The admin module does not have any dependency to assemblies or conferences, and in certain conditions may result in error.